src/Sarif.Viewer.VisualStudio/Views/CodeLocations.xaml

```diff
@@ -77,19 +73,22 @@
                                DataContext="{Binding Locations[0]}"
                                HorizontalAlignment="Stretch">
                         <DockPanel.ToolTip>
-                            <WrapPanel Orientation="Horizontal">
-                                <TextBlock>
-                                    <Run Text="{Binding FilePath, Mode=OneWay}" />
-                                </TextBlock>
-                                <TextBlock Visibility="{Binding Region.StartLine, Converter={StaticResource int32ToVisibilityConverter}}">
-                                    <Run Text=" Line" />
-                                    <Run Text="{Binding Region.StartLine, Mode=OneWay}" />
-                                </TextBlock>
-                                <TextBlock Visibility="{Binding Region.StartColumn, Converter={StaticResource int32ToVisibilityConverter}}">
-                                    <Run Text=" Col" />
-                                    <Run Text="{Binding Region.StartColumn, Mode=OneWay}" />
-                                </TextBlock>
-                            </WrapPanel>
+                            <ToolTip
+                                DataContext="{Binding PlacementTarget.DataContext, RelativeSource={RelativeSource Self}}">
+                                <WrapPanel Orientation="Horizontal">
```

Comment:
The property `FrameworkElement.ToolTip` is of type `object` -- it can be anything at all. But by wrapping the `WrapPanel` in a `ToolTip` object, we can make use of the property `ToolTip.PacementTarget` -- which specifies the UI element to which the tooltip is attached -- to access the `DockPanel`'s `DataContext`.